bot/exts/help_channels/_message.py

```diff
@@ -92,6 +93,38 @@ async def is_empty(channel: discord.TextChannel) -> bool:
     return False
 
 
+async def dm_on_open(message: discord.Message) -> None:
+    """
+    DM claimant with a link to the claimed channel's first message, with a 100 letter preview of the message.
+
+    Does nothing if the user has DMs disabled.
+    """
+    embed = discord.Embed(
+        title="Help channel opened",
+        description=f"You claimed {message.channel.mention}.",
+        colour=bot.constants.Colours.bright_green,
+        timestamp=message.created_at,
+    )
+
+    embed.set_thumbnail(url=constants.Icons.green_questionmark)
+    embed.add_field(
+        name="Your message", value=truncate_message(message, limit=100), inline=False
```

Comment:
```suggestion
        name="Your message:", value=truncate_message(message, limit=100), inline=False
```

Reply:
A field makes it render pretty fine without one.